src/std/error.rs

```diff
@@ -0,0 +1,42 @@
+use crate::std::rpc::XtStatus;
+pub use ac_node_api::metadata::{InvalidMetadataError, MetadataError};
+
+pub type ApiResult<T> = Result<T, Error>;
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
Everything was kept the same (simply copy pasted it) except for the Error `Other(#[from] Box<dyn std::error::Error + Send + Sync + 'static>),` Where I added `+ Send + Sync + 'static`

Reply:
Ok, I am not 100% sure, if this is satisfied by most errors, but I guess you could just do

```
ApiClientError::Other(e.to_string().into())
```
So fine with me.